tests/test_vis/test_vis_smpl.py

```diff
@@ -30,7 +30,7 @@ def test_visualize_smpl_pose():
         visualize_smpl_pose(
```

Comment:
Can we add a brief comment above each test to indicate its purpose? (i.e. what error it tries to catch)

Reply:
solved